model/fv_arrays.F90

```diff
@@ -1060,6 +1062,16 @@ module fv_arrays_mod
      logical :: BCfile_ne_is_open=.false.
      logical :: BCfile_sw_is_open=.false.
   end type fv_nest_type
+  
+  !3D-SA-TKE (kyf) (modify for data structure)
+  type sa3dtke_type
```

Comment:
Thank you for defining this useful datatype! It will be a good foundation for future development of 3D turbulence schemes.